openbas-front/src/admin/components/payloads/PayloadForm.tsx

```diff
@@ -161,6 +163,8 @@ const PayloadForm: FunctionComponent<Props> = ({
     handleSubmit(onSubmit)(e);
   };
 
+  const updateArchitecture = (type === 'Executable') || (type === 'Command');
```

Comment:
Question: I assume this means setting an architecture is mandatory?

Reply:
Yes, indeed. For architecture, we cannot set a null value for any type, but we can update the command and executable payloads. For other types, we set a default value.